Readme.rst

```diff
@@ -2370,6 +2449,16 @@ Examples::
     create-items bar CREATURE:CAT:SOAP
     create-items bar adamantine
 
+deathcause
+==========
+Focus a body part ingame, and this script will display the cause of death of
+the creature.
+Also works when selecting units from the (``u``) unitlist viewscreen.
+
+dfstatus
+========
```

Comment:
There isn't a `dfstatus` script in the scripts folder in the master or develop branches (only `gui/dfstatus`).